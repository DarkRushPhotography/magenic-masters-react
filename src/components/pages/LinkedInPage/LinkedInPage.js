import React, { Component } from "react";

import PageHeader from "../../@shared/PageHeader/PageHeader";
import Page from "../../@shared/Page/Page";

class LinkedInPage extends Component {

  render() {
    return (
      <Page>
        <PageHeader />
        <h1>LinkedIn</h1>
      </Page>
    );
  }
}

export default LinkedInPage;
