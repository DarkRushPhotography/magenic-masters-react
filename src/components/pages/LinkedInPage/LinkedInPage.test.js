import React from "react";
import { shallow } from "enzyme";

import LinkedInPage from "./LinkedInPage";

describe("LinkedInPage", () => {
  it("should render correctly", () => {
    const component = shallow(<LinkedInPage />);
    expect(component).toMatchSnapshot();
  });
});