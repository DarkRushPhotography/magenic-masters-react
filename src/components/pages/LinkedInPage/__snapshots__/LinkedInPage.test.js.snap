// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkedInPage should render correctly 1`] = `
<Page>
  <withRouter(WithStyles(PageHeader)) />
  <h1>
    LinkedIn
  </h1>
</Page>
`;
